Library/Formula/goolabs.rb

```diff
@@ -1,8 +1,8 @@
 class Goolabs < Formula
   desc "Command-line tool for morphologically analyzing Japanese language"
   homepage "https://pypi.python.org/pypi/goolabs"
-  url "https://pypi.python.org/packages/source/g/goolabs/goolabs-0.1.1.tar.gz"
-  sha1 "3201102ab0fad77d57a09dd7c879f6f163e05799"
+  url "https://pypi.python.org/packages/source/g/goolabs/goolabs-0.2.0.tar.gz"
+  sha1 "c544460b51ad63c7d79b0ef542816c2981fa22f9"
```

Comment:
@mikemcquiad  Sorry, I can't understand you. Did you say that I should change all sha1 to sha256?  

is this ok?
```
url "https://pypi.python.org/packages/source/g/goolabs/goolabs-0.2.0.tar.gz"
sha256 "34edd8b627322f39e9af0c220a196c493782e15fc302695d3701b4e3fbcc0101"  
```

thx